os/drivers/bluetooth/Make.defs

```diff
@@ -0,0 +1,45 @@
+############################################################################
```

Comment:
Please add Samsung License also.

Reply:
@jeongchanKim 
this is not my code.  It just moved to here.

Reply:
I think the license should be added here.